com.gooseberrydevelopment.pinepods.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gooseberrydevelopment.pinepods</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <name>Pinepods</name>
+  <summary>A complete, self-hosted podcast management system</summary>
+
+  <developer id="com.gooseberrydevelopment">
+    <name>Gooseberry Development</name>
+  </developer>
+
+  <description>
+    <p>
+      PinePods is a Rust based podcast management system that manages podcasts with multi-user support and relies on a central database with clients to connect to it. It's browser based and your podcasts and settings follow you from device to device due to everything being stored on the server. You can subscribe to podcasts and even hosts for podcasts with the help of the PodPeopleDB. It works on mobile devices and can also sync with a Nextcloud server or gpodder compatible sync server so you can use external apps like Antennapod as well! NOTE: This is the client edition of Pinepods. You must have a server to connect to in order to use this app.
+    </p>
+    <p>
+      Instructions for setting up your own self-hosted server can be found on the main repo
+    </p>
+  </description>
+  <launchable type="desktop-id">com.gooseberrydevelopment.pinepods.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/madeofpendletonwool/PinePods/blob/main/images/screenshots/homethemed.png?raw=true</image>
+      <caption>Main interface of Pinepods</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://github.com/madeofpendletonwool/PinePods</url>
+  <url type="bugtracker">https://github.com/madeofpendletonwool/PinePods/issues</url>
+  <url type="donation">https://github.com/sponsors/madeofpendletonwool</url>
+  <provides>
+    <binary>tauri-app</binary>
+  </provides>
```

Comment:
seems wrong? this should be the main binary of the app but desktop-applications don't need this.

Reply:
Removed this from the xml